File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int t=x,sum=0;
        while(x>0)
        {
            int y=x%10;
            sum=(sum*10)+y;
            x=x/10;
        }
        if(sum==t)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        # if(x<0):
        #     return False 
        # if(x==0):
        #     return True
        # n=""
        # s=str(x)
        # while(x!=0):
        #     t=x%10 
        #     n=n+str(t)
        #     x=x//10 
        # return s==n

        s=str(x)
        return s==s[::-1]